src/has.ts

```diff
@@ -11,4 +11,8 @@ add('dom-element-matches', function () {
 	}
 });
 
+add('dom-contextual-fragment', function() {
+	return !!(Range && Range.prototype.createContextualFragment);
```

Comment:
While I am guilty of this sometimes, the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md#general) says we should cast as `Boolean()` instead of `!!`.